server/install.py

```diff
@@ -68,13 +65,14 @@ def create_worker_log_dir():
 
 
 def install_all_testers():
-    settings = install_testers()
-    skeleton_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "autotest_server", "schema_skeleton.json")
-    with open(skeleton_file) as f:
-        skeleton = json.load(f)
-        skeleton["definitions"]["installed_testers"]["enum"] = list(settings.keys())
-        skeleton["definitions"]["tester_schemas"]["oneOf"] = list(settings.values())
-        REDIS_CONNECTION.set("autotest:schema", json.dumps(skeleton))
+    all_testers = list(TesterType)
+    installed_testers, installed_tester_settings = install_testers(all_testers)
```

Comment:
Returning the list `installed_testers` is redundant; you can obtain the list from the keys in `installed_tester_settings`.